stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -218,6 +220,22 @@ open class StripeEditText @JvmOverloads constructor(
         fun displayErrorMessage(message: String?)
     }
 
+    override fun onSaveInstanceState(): Parcelable {
+        return StripeEditTextState(
+            errorMessage,
+            shouldShowError,
+            super.onSaveInstanceState()
+        )
+    }
+
+    override fun onRestoreInstanceState(state: Parcelable?) {
+        (state as StripeEditTextState).let {
+            super.onRestoreInstanceState(it.superState)
+            errorMessage = it.errorMessage
+            shouldShowError = it.shouldShowError
+        }
```

Comment:
Do we need to call `super.onRestoreInstanceState(state)` if it's not a `StripeEditTextState`?

Reply:
hmm don't think so, per the [doc](https://developer.android.com/reference/android/view/View#onRestoreInstanceState(android.os.Parcelable)), it's always restoring from a result of `onSaveInstanceState` - and we are always returning `StripeEditTextState`